Makefile

```diff
@@ -0,0 +1,72 @@
+clean:
+	rm -rf bazel-*
+	rm -rf man
+	rm -rf docs
+
+build:
+	bazel build //...
+
+test:
+	bazel test //...
+
+.PHONY: clean build test
+
+bazel-bin/verilog/tools/lint/verilog_lint:
+	bazel build //verilog/tools/lint:verilog_lint
+
+bazel-bin/verilog/tools/syntax/verilog_syntax:
+	bazel build //verilog/tools/syntax:verilog_syntax
+
+bazel-bin/verilog/tools/formatter/verilog_format:
+	bazel build //verilog/tools/formatter:verilog_format
+
+tools: bazel-bin/verilog/tools/lint/verilog_lint bazel-bin/verilog/tools/syntax/verilog_syntax bazel-bin/verilog/tools/formatter/verilog_format
+	@true
+
+# Generate man pages from the tools
+man/verilog_lint.1: bazel-bin/verilog/tools/lint/verilog_lint
+	mkdir -p man
+	gflags2man --help_flag="--helpfull" --dest_dir man bazel-bin/verilog/tools/lint/verilog_lint
+
+man/verilog_syntax.1: bazel-bin/verilog/tools/syntax/verilog_syntax
+	mkdir -p man
+	gflags2man --help_flag="--helpfull" --dest_dir man bazel-bin/verilog/tools/syntax/verilog_syntax
+
+man/verilog_format.1: bazel-bin/verilog/tools/formatter/verilog_format
+	mkdir -p man
+	gflags2man  --help_flag="--helpfull" --dest_dir man bazel-bin/verilog/tools/formatter/verilog_format
+
+man/%.gz: man/%
+	gzip -k -f $<
+
+man: man/verilog_lint.1.gz man/verilog_syntax.1.gz man/verilog_format.1.gz
+	@true
+
+.PHONY: man
+
+DOCS_DIR=docs
+docs: docs.sh *.md
+	./docs.sh $(DOCS_DIR)
+
+.PHONY: docs
+
+PREFIX=/usr/local
```

Comment:
Maybe make `PREFIX?=/usr/local` ? That way, it can be overridden by an environment variable.

I'd probably put all these variable definitions at the top of the file.